lib/bolt/outputter/json.rb

```diff
@@ -125,10 +125,11 @@ def print_target_info(target_list, _inventoryfile)
         )
       end
 
-      def print_groups(groups)
+      def print_groups(groups, inventoryfile)
         count = groups.count
         @stream.puts({ groups: groups,
-                       count: count }.to_json)
+                       count: count,
+                       inventoryfile: inventoryfile.to_s }.to_json)
```

Comment:
In `bolt target show` output this is nested under `inventory: { file: }` I think following that same structure here would good for consistency and listing the groups there as well.

Reply:
The `inventory` section in JSON is meant for showing which targets are loaded from inventory. I'm not sure that really applies here, since all groups (except `all`) _must_ come from inventory. Removed adding the inventory file for now and can be addressed in a separate ticket.

Reply:
Consistent JSON API response formats are a [pretty basic tenet](https://stackoverflow.com/questions/50148694/rest-api-json-format-consistency) of API design so that users can predict and understand responses easily without having to fail first, run the command manually and study the output, and adjust. I was actually thinking we should also have the `groups` listed both under `inventory` and at the top level similar to `inventory show` output as well for this reason. Plus, *not* all groups are defined in the inventory file: the 'all' group is a notable exception, and if users are loading inventory from an environment variable I think that'd probably be a different key under the 'inventory' key.

Don't take it from me, here are some other sources:
- [“consistency underlies all principles. A good architecture is consistent in the sense that, given a partial knowledge of the system, one can predict the remainder.”](https://cloud.google.com/blog/products/api-management/api-design-best-practices-common-pitfalls)
- ["It should be simple, intuitive, and consistent to make adoption not only easy but pleasant"](https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api)
- ["it’s best to work toward consistency when designing new aspects of an existing API."](https://www.oreilly.com/content/best-design-practices-to-get-the-most-out-of-your-api/)

No one's asked for this specifically yet so it's fine to leave it out so we don't paint ourselves into a corner.